node/src/main/scala/coop/rchain/node/effects/ReplClient.scala

```diff
@@ -43,15 +46,18 @@ class GrpcReplClient(host: String, port: Int, maxMessageSize: Int)
       .attempt
       .map(_.leftMap(processError))
 
-  def eval(fileNames: List[String]): Task[List[Either[Throwable, String]]] =
+  def eval(
+      fileNames: List[String],
+      unmatchedSends: Boolean
+  ): Task[List[Either[Throwable, String]]] =
     fileNames
-      .traverse(eval)
+      .traverse(eval(_, unmatchedSends))
 
-  def eval(fileName: String): Task[Either[Throwable, String]] = {
+  def eval(fileName: String, printUnmatchedSendsOnly: Boolean): Task[Either[Throwable, String]] = {
```

Comment:
should this private?

Reply:
Nope, it's called from `ReplClient.scala`.